File: Programmers/Python/시저 암호.py
def solution(s, n):
    answer = ''
    for i in s:
        if "a"<= i <="z":
            if chr(ord(i)+n) > "z":
                answer += chr(ord(i)+n-26)
            else :
                answer += chr(ord(i)+n)
        elif "A"<= i <="Z":
            if chr(ord(i)+n) > "Z":
                answer += chr(ord(i)+n-26)
            else :
                answer += chr(ord(i)+n)
        else :
            answer += i
    return answer

File: Programmers/Java/Level 1/시저 암호.java
class Solution {
    public String solution(String s, int n) {
        String answer = "";
        for(int i=0; i<s.length(); i++){
            char c = s.charAt(i);
            
               if(s.charAt(i)>='A' && s.charAt(i)<='Z'){
                   if(c+n>'Z'){
                       answer += (char)(c+n-26);
                   }
                   else
                       answer += (char)(c+n);
               }
                else if(s.charAt(i)>='a' && s.charAt(i)<='z'){
                    if(c+n>'z'){
                       answer += (char)(c+n-26);
                   }
                   else
                       answer += (char)(c+n);
                }
                else
                    answer += (char)c;
            }
                    
        return answer;
    }
}

// 참고